### PR TITLE
completed sidebar feature

### PR DESCRIPTION
**Pull request by Tanvir04khan**

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -6,12 +7,19 @@ import SideBar from "./components/Home/sideBar/SideBar";
...
 
 const App = () => {
+  const [isSidebarVisible, setIsSidebarVisible] = useState(false);
+
+  const handleToggleSidebar = () => setIsSidebarVisible((value) => !value);
+
   return (
     <>
-      <Header />
+      <Header handleToggleSidebar={handleToggleSidebar} />
```
Comment (faran4engg) [inline src/App.jsx]: Prop name should be `onToggleSidebar`

File: src/App.jsx [hunk 2]
```diff
@@ -6,12 +7,19 @@ import SideBar from "./components/Home/sideBar/SideBar";
...
       <div className="app__container">
-        <SideBar />
-        <Container fluid className="app__main border border-warning">
+        <SideBar
+          isSidebarVisible={isSidebarVisible}
+          handleToggleSidebar={handleToggleSidebar}
```
Comment (faran4engg) [inline src/App.jsx]: Same here

File: src/components/Home/header/Header.jsx [hunk 3]
```diff
@@ -6,10 +6,14 @@ import { FaBars } from "react-icons/fa";
 import { AiOutlineSearch } from "react-icons/ai";
 import { MdNotifications, MdApps } from "react-icons/md";
 
-const Header = () => {
+const Header = ({ handleToggleSidebar }) => {
```
Comment (faran4engg) [inline src/components/Home/header/Header.jsx]: This will be changed as a part of above comments

File: src/components/Home/sideBar/SideBar.jsx [hunk 4]
```diff
@@ -1,8 +1,50 @@
...
+  MdHome,
+  MdSentimentDissatisfied,
+} from "react-icons/md";
 
-const Sidebar = () => {
-  return <div className="border border-danger">sidebar</div>;
+const Sidebar = ({ isSidebarVisible, handleToggleSidebar }) => {
+  return (
+    <div className={!isSidebarVisible ? "sidebar" : "sidebar open"}>
+      <li onClick={handleToggleSidebar}>
```
Comment (faran4engg) [inline src/components/Home/sideBar/SideBar.jsx]: As its temporary, instead if adding it on every li, you add it to the div 😃

**Review by faran4engg — APPROVED** [review]
Minor changes. Approving already

File: src/App.jsx (hunk 1 above)
Comment (Tanvir04khan) [reply src/App.jsx]: changed prop name as per suggestion.

File: src/App.jsx (hunk 2 above)
Comment (Tanvir04khan) [reply src/App.jsx]: changed prop name as per suggestion.

File: src/components/Home/header/Header.jsx (hunk 3 above)
Comment (Tanvir04khan) [reply src/components/Home/header/Header.jsx]: changed prop name as per suggestion.

File: src/components/Home/sideBar/SideBar.jsx (hunk 4 above)
Comment (Tanvir04khan) [reply src/components/Home/sideBar/SideBar.jsx]: changed prop name as per suggestion.

**Pull request merged**